packages/roc-package-webpack-web-dev/src/config/roc.config.js

```diff
@@ -3,7 +3,8 @@ const config = {
         dev: {
             devMiddleware: {
                 noInfo: true,
-                quiet: false
+                quiet: false,
+                poll: false
```

Comment:
Should we maybe support defining a custom interval, not just the default?
https://github.com/webpack/watchpack#api

Have we made any performance tests with this, maybe we need to also use `aggregateTimeout `?